### PR TITLE
feat: Properties

### PR DESCRIPTION
**Pull request by keithlogan94**
# Properties

- [x] You've read the [Contributor Guide](https://github.com/recro/medulla/blob/main/CONTRIBUTING.md) and [Code of Conduct](https://github.com/recro/medulla/blob/main/CODE_OF_CONDUCT.md).
- [x] You've included unit or integration tests for your change, where applicable.
- [x] You've included inline docs for your change, where applicable.
- [x] There's an open issue for the PR that you are making. If you'd like to propose a new feature or change, please open an issue to discuss the change or find an existing issue.

# Properties Window 

### This will be a single place to edit any properties for medulla.

Fixes #125

### CODE REVIEW

File: src/Editor/Client/Components/EditorPage.razor [hunk 1]
```diff
@@ -464,11 +474,11 @@
...
-    
-    
-    
-    
-    
+
+
+
+
+
```
Comment (AndrewBabbitt97) [inline src/Editor/Client/Components/EditorPage.razor:481]: So these were white spaces, lets just clean this up

File: src/Editor/Client/Components/EditorPage.razor [hunk 2]
```diff
@@ -1532,12 +1542,12 @@
...
+
```
Comment (AndrewBabbitt97) [inline src/Editor/Client/Components/EditorPage.razor:1550]: Same thing here for white spaces, just remove them instead of adding

**Review by AndrewBabbitt97 — COMMENTED** [review]
LGTM aside from the minor oddity with the newlines / whitespace. Remove that and were good for merge o/

File: src/Editor/Client/Components/EditorPage.razor (hunk 1 above)
Comment (keithlogan94) [reply src/Editor/Client/Components/EditorPage.razor:481]: Sounds good

**Review by AndrewBabbitt97 — APPROVED** [review]
LGTM

**Pull request merged**